Source/WebKit/Platform/cocoa/CocoaHelpers.h

```diff
@@ -75,12 +77,29 @@ T *objectForKey(const RetainPtr<NSDictionary>& dictionary, id key, bool returnin
     return objectForKey<T>(dictionary.get(), key, returningNilIfEmpty, containingObjectsOfClass);
 }
 
+// MARK: NSDictionary helper methods.
+
+NSDictionary *dictionaryWithLowercaseKeys(NSDictionary *);
+NSDictionary *mergeDictionaries(NSDictionary *, NSDictionary *);
+NSDictionary *mergeAndSetValues(NSDictionary *, NSDictionary *);
```

Comment:
Either name the first one `merge()` or the second one `mergeDictionariesAndSetValues()`.